pkg/logging/init.go

```diff
@@ -168,7 +170,7 @@ func getHandler(w io.Writer, encoding Encoding, hopts slog.HandlerOptions) (th s
 	case EncodingJSON:
 		th = slog.NewJSONHandler(w, &hopts)
 	case EncodingLogfmt:
-		th = slog.NewJSONHandler(w, &hopts)
+		th = slog.NewTextHandler(w, &hopts)
```

Comment:
Thx, now the output looks much better on the console, didn't see that! :rofl:

Reply:
Yeah. I'm glad you opened this one up as a dedicated issue because I fixed that drive-by in the long-running branch #120 . A good reminder for me not to mix things too much in a PR and rather rebase frequently.